src/components/sideNavigation/SideNavigationLoadingHeader.tsx

```diff
@@ -0,0 +1,7 @@
+import * as React from 'react';
```

Comment:
unless i'm wrong you don't even need to import react here

Reply:
If we don't import React, we get this error:

![image](https://user-images.githubusercontent.com/5606626/34627585-26f70784-f22f-11e7-8866-a297d4b79d5c.png)